common/src/main/java/io/druid/metadata/MetadataStorageActionHandler.java

```diff
@@ -104,9 +104,8 @@ public void insert(
    * Remove the lock with the given lock id.
    *
    * @param lockId lock id
-   * @return true if the lock was removed, false if the given lock id did not exist
    */
-  public boolean removeLock(long lockId);
+  public void removeLock(long lockId);
```

Comment:
Probably not, I don't think it matters if the lock was actually removed. We just want it to be gone.